pypeit/find_objects.py

```diff
@@ -166,8 +166,19 @@ def __init__(self, sciImg, slits, spectrograph, par, objtype, wv_calib=None, wav
         # Internal bpm mask
         # We want to keep the 'BOXSLIT', which has bpm=2. But we don't want to keep 'BOXSLIT'
         # with other bad flag (for which bpm>2)
-        self.reduce_bpm = (self.slits.mask > 2) & (np.invert(self.slits.bitmask.flagged(
-                        self.slits.mask, flag=self.slits.bitmask.exclude_for_reducing)))
+        # TODO: To my mind, we should never be using the value of the bit to
```

Comment:
@kbwestfall I tested this and it looks OK.

Her is the test case, in case you want to add it to the tests:

```
>>> import numpy
>>> from pypeit.slittrace import SlitTraceBitMask
>>> bm = SlitTraceBitMask()

>>> v = numpy.array([10,0,2])

>>> print([bm.flagged_bits(_v) for _v in v])
[['BOXSLIT', 'BADWVCALIB'], [], ['BOXSLIT']]

>>> desired_bpm = (v > 2) & (np.invert(bm.flagged(v, flag=bm.exclude_for_reducing)))
>>> desired_bpm
array([ True, False, False])

>>> new_bpm = bm.flagged(v, exclude='BOXSLIT', and_not=bm.exclude_for_reducing)
>>> new_bpm
array([ True, False, False])

>>> np.all(new_bpm == desired_bpm)
True
```

Reply:
Thanks!  I added this test to `test_bitmask.py`.